plugins/sfIsaarPlugin/modules/sfIsaarPlugin/actions/relatedAuthorityRecordComponent.class.php

```diff
@@ -93,7 +93,7 @@ protected function processField($field)
                 $value = $this->form->getValue('resource');
                 if (isset($value)) {
                     $params = $this->context->routing->parse(Qubit::pathInfo($value));
-                    if ($this->resource->id != $this->relation->objectId) {
+                    if ($this->resource->id != $this->relation->objectId || $this->resource->id == null) {
```

Comment:
Not a big deal, but I think the following is clearer ...

```php
if (!isset($this->resource->id) || $this->resource->id != $this->relation->objectId) {
```

Not in this case, but usually checking the existency before comparing may prevent some issues.